test/ir_Corona_test.cpp

```diff
@@ -1385,21 +1385,28 @@ TEST(TestCoronaAcClass, Power) {
 
   ac.on();
   EXPECT_TRUE(ac.getPower());
+  EXPECT_TRUE(ac.getPowerButton());
 
   ac.off();
   EXPECT_FALSE(ac.getPower());
+  EXPECT_TRUE(ac.getPowerButton());
 
   ac.setOnTimer(60);
+  EXPECT_FALSE(ac.getPowerButton());
   ac.setPower(true);
   EXPECT_TRUE(ac.getPower());
   EXPECT_EQ(0, ac.getOnTimer());
+  EXPECT_TRUE(ac.getPowerButton());
 
   ac.setOffTimer(60);
+  EXPECT_FALSE(ac.getPowerButton());
   ac.setPower(false);
   EXPECT_FALSE(ac.getPower());
   EXPECT_EQ(0, ac.getOffTimer());
+  EXPECT_TRUE(ac.getPowerButton());
 
   ASSERT_EQ(4, kCoronaAcPowerOffset);
+  ASSERT_EQ(5, kCoronaAcPowerButtonOffset);
```

Comment:
Wow. I didn't notice this before. That's awesome belt **&** suspenders stuff. ;-)

Reply:
I would have been surprised if you did notice, the rename to kCoronaAcPowerButtonOffset and the add of that ASSERT was added in the last commits, just pushed a few minutes ago ;D